torch/_dynamo/output_graph.py

```diff
@@ -660,7 +660,7 @@ def add_symbol_bindings(self, arg: GraphArg):
 
         assert arg.fake_tensor is not None
 
-        def bind_symint(s, prop):
+        def bind_symint(s: torch.SymInt, prop):
```

Comment:
nit specify return type

Reply:
There are actually a lot of untyped defs, I'm working on them in the next PR in stack